s3-sanity-test.sh

```diff
@@ -45,20 +45,7 @@ fi
 test_file_input=/tmp/SanityObjectToDeleteAfterUse.input
 test_output_file=/tmp/SanityObjectToDeleteAfterUse.out
 
-ldappasswd=""
-if rpm -q "salt"  > /dev/null;
-then
-    ldappasswd=$(salt-call pillar.get openldap:iam_admin:secret --output=newline_values_only)
-    ldappasswd=$(salt-call lyveutil.decrypt openldap ${ldappasswd} --output=newline_values_only)
-else
-    # Dev environment. Read ldap admin password from "/root/.s3_ldap_cred_cache.conf"
```

Comment:
I am not sure on the answer.

What I know is that we need to do code changes (someone might have also started that) in init.sh script to install py-utils on dev VMs so that s3cipher can be used for ldappasswords.

I guess we can remove the code to ask ldappassword in the ansible script.

Reply:
yes.. currently Vimal is doing changes to install this cortx-py-utils package but i guess we need to remove  reference which are using ldappassword from user
cc @nileshgovande

Reply:
yes, but that is possible only when we have the "install py-utils on dev VM" story done.

Reply:
yes.